pages/about-us/index.tsx

```diff
@@ -0,0 +1,151 @@
+import Head from 'next/head'
```

Comment:
General thoughts: same comment regarding multiple ternarys for language check as Success Stories, it's easier to do one check and then load that.